assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -100,6 +100,7 @@ function DropDown({
   lightMode = false,
   placement = Placement.BOTTOM_END,
   placeholder = __('Select an option', 'web-stories'),
+  disabledOptions = [],
```

Comment:
💭 should we add `disabled` as a property on each option rather than adding them as separate arrays?

Reply:
ohh, like as part of an option in the options prop being passed in here?

Reply:
Yeah exactly! That way you don't have to iterate over the disabled options when the option itself would just tell you if it is disabled

Reply:
smart. I like it. will see if it's easy to move there